lib/android/src/main/java/com/wix/interactable/physics/PhysicsAnimator.java

```diff
@@ -79,6 +79,17 @@ public void addBehavior(PhysicsBehavior behavior) {
         ensureRunning();
     }
 
+    public void removeBehavior(PhysicsBehavior behavior) {
```

Comment:
It might make sense to return a bool value whether the removal was successful or not.

Reply:
Sorry, I was AFK yesterday.